applications/Chat/evaluate/README.md

```diff
@@ -1,182 +1,217 @@
+* `target` (str, optional): The target answer for the instruction.
+* `id` (int, compulsory): The ID of the instruction / question.
+
+Example:
+```
+[
+    {
+        "category": "brainstorming",
+        "instruction": "请介绍一下人工智能的多个领域。",
+        "input": "",
+        "output": "{GPT-3.5 Answers}",
+        "target": "{Model Answers / Predictions}",
+        "id": 1
+    },
+    {
+        "category": "classification",
+        "instruction": "新闻标题：为什么电影《倩女幽魂》中燕赤霞一个道士却拿着金刚经？请根据新闻标题判断新闻所属的分类，你需要从文化，娱乐，体育，财经，房产，教育，科技，旅游，游戏，军事这十类中选择一个答案。",
+        "input": "",
+        "output": "{Model Answers / Predictions}",
+        "target": "{target answer}",
+        "id": 2
+    }
+]
+```
+
+### Evaluation
+#### Configuration
+The configuration file `config_cn.json` can control how evaluate the performance of the model.
+The following is an example showing the config structure:
+```
+{
+    "language": "cn",
+    "category": {
+        "brainstorming": {
+            "GPT-3.5": ["relevance", "creativity", "practicality", "correctness"],
+            "GPT-4": [],
+            "Metrics": ["Distinct"]
+        },
+        "chat": {
+            "GPT-3.5": [ "relevance", "naturalness", "engagingness", "reasonableness"],
+            "GPT-4": [],
+            "Metrics": ["Distinct"]
+}
+```
+`"language"`: evaluate the model capability in which language, choosing from `"cn"` (we only support Chinese for now).
+`"category"`: evaluate the model capability in which category/categories, choosing from `"brainstorming"`,`"chat"`,`"classification"`,`"closed_qa"`,`"extraction"`,`"generation"`,`"open_qa"`,`"rewriting"`,`"roleplay"`,`"summarization"`.
+`"GPT-3.5"`: config metrics for GPT-3.5 evaluation, choosing from `"language organization"`,`"relevance"`,`"creativity"`,`"practicality"`,`"correctness"`,`"naturalness"`,`"engagingness"`,`"reasonableness"`,`"diversity"`,`"fidelity"`,`"conciseness"`.
+`"GPT-4"`: config metrics for GPT-4 evaluation, choosing from `"language organization"`,`"relevance"`,`"creativity"`,`"practicality"`,`"correctness"`,`"naturalness"`,`"engagingness"`,`"reasonableness"`,`"diversity"`,`"fidelity"`,`"conciseness"`.
+`"Metrics"`: config metrics for automatic metrics evaluation, choosing from `"BLEU"`,`"ROUGE"`,`"Distinct"`,`"BERTScore"`,`"Precision"`,`"Recall"`,`"F1 score"`.
+
+#### Evaluate
+After setting the configuration file, you can evaluate the model using `eval.py`.
+
+
+An example script is provided as follows:
+```shell
+python eval.py \
+    --config_file "path to the config file" \
+    --battle_prompt_file "path to the prompt file for battle" \
+    --gpt_evaluation_prompt_file "path to the prompt file for gpt evaluation" \
+    --target_file "path to the target answer file" \
+    --answer_file_list "path to the answer files of at most 2 models" \
+    --model_name_list "the names of at most 2 models" \
+    --save_path "path to save results" \
+    --openai_key "your openai key" \
+```
+
```

Comment:
You can add a todo list for the following tasks:

1. Evaluation standard for English
2. Support UniEval
3. Support GPT-4 evaluation